nixos/modules/misc/ids.nix

```diff
@@ -336,6 +336,7 @@
       solr = 309;
       alerta = 310;
       minetest = 311;
+      bitcoin = 333;
```

Comment:
why 333 instead of 312 here?

Reply:
@jb55 Because `bitcoind` default ports all end with 333, and 333 was below the cap of 399

Reply:
This is the user id, has nothing to do with ports.

Reply:
Shouldn't bitcoind work without static UID/GID and systemd Dynamic User ?